modules/form-field/css/lib/form-field.scss

```diff
@@ -5,6 +5,9 @@
 
   .wdc-form-field {
     position: relative;
+    min-height: 32px;
```

Comment:
There are a lot of magic numbers like this throughout our CSS code. Some CSS has addition between spacing values. Other values don't have and spacing match. Is there rationale behind when we choose values?

Reply:
+1. We should always use the spacing values if possible, and if we're doing additions, we should denote the addition happening, rather than just using the result.